website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx

```diff
@@ -38,6 +38,8 @@ To delete a key, find it in the list of keys and click its "Delete" button. Befo
 
 ~> **Important:** If any workspaces are still using a key when you delete it, they will be unable to clone modules from private repos until you assign them a new key. This might cause Terraform runs to fail.
 
+~> **Important:** If you upload an invalid ssh key, after uploading the correct ssh key, you will want to push a new commit for the new ssh key to take effect.
```

Comment:
```suggestion
~> **Important:** If you upload an invalid ssh key, you should upload the correct ssh key and push a new commit for the new key to take effect.
```

A slight suggestion for change to use fewer words, avoid future tense, and for clarity.